documentation/assemblies/security/assembly-security.adoc

```diff
@@ -8,7 +8,12 @@
 [role="_abstract"]
 Strimzi supports TLS for encrypted communication between Kafka and Strimzi components.
 
-Communication between Kafka brokers (interbroker communication), between ZooKeeper nodes (internodal communication), and between those components and the Strimzi operators is always encrypted.
+Communication is always encrypted for communication between the following components:
+
+* Interbroker communication between Kafka brokers
+* Internodal communication between ZooKeeper nodes
+* Strimzi operator communication with Kafka brokers and ZooKeeper nodes
```

Comment:
Should this add also communication between Kafka and ZooKeeper? That is always encrypted as well.